packages/utils/package.json

```
@@ -29,9 +29,10 @@
     "dist"
   ],
   "scripts": {
-    "pretest": "pnpm fuels-forc build -p test/forc-projects/simple",
+    "pretest": "pnpm build:forc",
     "build": "tsup",
-    "postbuild": "tsx ../../scripts/postbuild.ts"
+    "postbuild": "tsx ../../scripts/postbuild.ts",
+    "build:forc": "pnpm fuels-forc build -p test/forc-projects/simple"
```

Same nit again, apologies.

```suggestion
    "build:forc": "pnpm fuels-forc build -p test/forc-projects/simple"
    "postbuild": "tsx ../../scripts/postbuild.ts",
```